scio-core/src/main/java/com/spotify/scio/util/RemoteFileUtil.java

```diff
@@ -69,13 +69,14 @@ public Path load(URI key) throws Exception {
               });
 
   /** Create a new {@link RemoteFileUtil} instance. */
-  public static RemoteFileUtil create(PipelineOptions options) {
+  private RemoteFileUtil() {}
+
+  public static void configure(PipelineOptions options) {
```

Comment:
Why not calling `FileSystems.setDefaultPipelineOptions(options)` directly ?
I see we have this already in some places

Reply:
like as part of a static initialization? we could do that -- my thought process was that we don't want `FileSystems.setDefaultPipelineOptions(options)` to be called on DoFn deserialization (on the worker), only at graph construction time, right? I guess it doesn't matter too much, though.

Reply:
Maybe I miss smth, Isn't the `RemoteFileUtil` used in the transform step only ?
At the graph, we only require the file URI + deserialization method.
`RemoteFileUtil` is used on the worker to retrieve those files right ?